Source/EasyNetQ/Consumer/AsyncBasicConsumer.cs

```diff
@@ -72,24 +77,26 @@ ReadOnlyMemory<byte> body
         if (cts.IsCancellationRequested)
             return;
 
+        var messageReceivedInfo = new MessageReceivedInfo(
+            consumerTag, deliveryTag, redelivered, exchange, routingKey, queue.Name
+        );
+        var messageBody = body;
+        var messageProperties = new MessageProperties();
+        messageProperties.CopyFrom(properties);
+
         if (logger.IsDebugEnabled())
         {
             logger.DebugFormat(
-                "Message delivered to consumer {consumerTag} with deliveryTag {deliveryTag}",
-                consumerTag,
-                deliveryTag
+                "Message with properties {@properties} delivered to consumer {consumerId} with receivedInfo {@receivedInfo}",
```

Comment:
Changing field type from scalar to object is breaking change for logging consumers like ELK since this leads to field conflicts in existing indices. LogStash often cannot parse incoming json document and just discards it. It is very annoying in production.